unit_tests/test_cli.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import webbrowser
+import os
+import shutil
+from robyn.__main__ import create_robyn_app, docs
+
+
+class TestCreateRobynApp(unittest.TestCase):
+    def setUp(self) -> None:
+        self.test_directory_name = "app1"
+        self.is_docker_needed_yes = "Y"
+        self.is_docker_needed_no = "N"
+
+    def tearDown(self) -> None:
+        if os.path.exists(self.test_directory_name):
+            shutil.rmtree(self.test_directory_name)
```

Comment:
correct me if i am wrong, we're creating a temp directory "app1" in unit_tests/* ? 

I am not sure if this is the write a way to go about it?

I suggest we can use https://docs.python.org/3/library/tempfile.html to create a temporary directory instead.

This would in return ensure automatic cleanup and probably safer to use?

Thoughts?

Reply:
Hey @banipreetr, So we are updating the current test and converting into mocking based approach, so it'll ensure all this things.

Reply:
Thanks for the reply @Mr-Sunglasses, but coming back to my main question.
-> Is this test trying to create an "app1" directory?
    -> If yes, what is the location of that directory?